src/BreadcrumbItem.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import classNames from 'classnames';
+import BootstrapMixin from './BootstrapMixin';
+import SafeAnchor from './SafeAnchor';
+
+const BreadcrumbItem = React.createClass({
+  mixins: [BootstrapMixin],
+
+  propTypes: {
+    linkId: React.PropTypes.string,
+    href: React.PropTypes.string,
+    title: React.PropTypes.node,
+    target: React.PropTypes.string
+  },
+
+  render() {
+    const {
+      href,
+      title,
+      target,
+      linkId,
+      children,
+      ...props } = this.props;
+    const active = (href == null);
```

Comment:
Actually I also prefer to set the `active` prop explicitly, but I don't know how to properly handle `active` and `href` in these situations: 
1. if `active` is false and `href` is null: should I remove the `href` prop?
2. if `active` is true and `href` is given as well: [in bootstrap](http://getbootstrap.com/components/#breadcrumbs), an `active <li>` element doesn't have an `anchor` element. Is there any other good place to put the `href` prop, or just add an `anchor` to the `active <li>` element?